js/src/modals/CreateAccount/store.js

```diff
@@ -220,7 +226,28 @@ export default class Store {
     this.stage--;
   }
 
-  createAccount = () => {
+  createAccount = (vaultStore) => {
```

Comment:
No. We specifically don't want it in FirstRun. So it is an option, not a requirement.